primitives/src/calendar.rs

```diff
@@ -707,11 +425,16 @@ pub struct CalendarPreviousMonthButtonProps {
 #[component]
 pub fn CalendarPreviousMonthButton(props: CalendarPreviousMonthButtonProps) -> Element {
     let ctx: CalendarContext = use_context();
+    // disable previous button when we reach the limit
+    let mut button_disabled = use_signal(|| false);
     // Handle navigation to previous month
     let handle_prev_month = move |e: Event<MouseData>| {
         e.prevent_default();
         let current_view = (ctx.view_date)();
-        ctx.set_view_date.call(current_view.prev_month());
+        match current_view.checked_sub_months(Months::new(1)) {
+            Some(prev_month) => ctx.set_view_date.call(prev_month),
```

Comment:
Just to be sure. If we have reached the limit (`button_disabled` set true) and then started moving through months in reverse direction, will the button return?